packages/design-system/src/ToastLegacy.stories.tsx

```
@@ -0,0 +1,161 @@
+export const Warning = () => {
+  return (
+    <div>
+      <div>Styled toast component with warning type.</div>
+      <ToastLegacy triggerText="Trigger" type="warning">
+        Content of Tooltip
+      </ToastLegacy>
+    </div>
+  )
+}
+
+export const Error = () => {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)